flotta-agent.spec

```diff
@@ -38,12 +38,17 @@ go build -o ./bin/device-worker ./cmd/device-worker
 cd flotta-agent-%{VERSION}
 mkdir -p %{buildroot}%{_libexecdir}/yggdrasil/
 install ./bin/device-worker %{buildroot}%{_libexecdir}/yggdrasil/device-worker
+make install-worker-config LIBEXECDIR=%{_libexecdir} BUILDROOT=%{buildroot} SYSCONFDIR=%{_sysconfdir}
 
 %files
 %{_libexecdir}/yggdrasil/device-worker
+%{_sysconfdir}/yggdrasil/
 
 %changelog
 
+* Fri Feb 11 2022 Eloy Coto <eloycoto@acalustra.com> 1.1
```

Comment:
Does it mean we want to release this version?

Reply:
No, just to keep track of changes in the RPM specfile.